### PR TITLE
Bumps the crd version

### PR DESCRIPTION
**Pull request by 0sewa0**

### CODE REVIEW

File: config/openshift copy/kustomization.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```
Comment (chrismuellner) [inline config/openshift copy/kustomization.yaml:1]: Is the `openshift copy` folder still needed?

File: config/openshift3.11/kustomization.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```
Comment (chrismuellner) [inline config/openshift3.11/kustomization.yaml:1]: Would an `openshift-common` folder with patches for the mismatched apiversions work for this?

similar to the following folder structure:
- openshift-common
	- `mutatingwebhookconfiguration.yaml`
	- `securitycontextconstraint.yaml`
	- ...
- openshift
	- kustomization with `common` + `openshift-common`
	- default CRD
- openshift 3.11
	- kustomization with `common` + `openshift-common`
	- patch to change apiversion of `mutatingwebhookconfiguration.yaml` to `v1beta1` 
	- openshirt 3.11 CRD

**Review by chrismuellner — CHANGES_REQUESTED** [review]

File: config/openshift copy/kustomization.yaml (hunk 1 above)
Comment (0sewa0) [reply config/openshift copy/kustomization.yaml:1]: how was that not deleted is a mystery to me 😅

File: Makefile [hunk 3]
```diff
@@ -58,9 +59,12 @@ deploy: manifests kustomize
 
 # Generate manifests e.g. CRD, RBAC etc.
 manifests: controller-gen
-	$(CONTROLLER_GEN) $(CRD_OPTIONS) rbac:roleName=manager-role webhook paths="./..." output:crd:artifacts:config=config/crd/bases
+	$(CONTROLLER_GEN) $(CRD_OPTIONS) rbac:roleName=manager-role webhook paths="./..." output:crd:artifacts:config=config/crd/default/bases
 	python3 ./hack/customize_crds.py
 
+manifests-ocp311: controller-gen
```
[the remark this answers is not in the document]
Comment (0sewa0) [reply Makefile:65]: `kubectl apply -k ./config/openshift3.11`

**Review by chrismuellner — APPROVED** [review]

**Pull request merged**